src/animation.js

```diff
@@ -33,8 +33,8 @@ var _cancelFrame = function(inId) {
 };
 
 for (var i = 0, pl = prefix.length, p, wc, wr; (p = prefix[i]) || i < pl; i++) {
-	// if we're on ios 6 just use setTimeout, requestAnimationFrame has some kinks currently
-	if (platform.ios >= 6) {
+	// if we're on ios 6 just use setTimeout, requestAnimationFrame has some kinks
+	if (platform.ios < 7) {
```

Comment:
No, I mean exclusively for iOS 6. :smile: It doesn't change the ultimate result and these are old platform versions anyways, but I wanted to preserve the intent of the feature detection here, rather than start a trend of having blanket conditionals. iOS 6 is the exception case we need to explicitly handle because it actually has an implementation of `requestAnimationFrame` (and would pass our feature detection), but that implementation is quirky/buggy and does not work properly, which is why we added the conditional in the first place.